docs/Spark/extensibility/dependencies.md

```diff
@@ -15,158 +15,78 @@ to [Maven](https://mvnrepository.com/) or [PyPi](https://pypi.org/) coordinates.
+**pom.xml** file. If you're using Python, dependencies are saved in the **setup.py** file.
 
-Dependencies are natively saved within your build-system files. E.g. if you're using Scala, they're saved in the
-**pom.xml** file, if you're using Python, they're saved in the **setup.py** file. To customize the build system further,
-please check out the [Custom Build Tool](#build-tool-customization) section.
+## View dependencies
 
-## Common Actions
+There are two ways to view all of your Project dependencies:
 
-### See dependencies
+- Open your Project from the **Metadata** page. Then, click on the **Dependencies** tab.
+- Open your Project in the **Pipeline Editor**. Then, click **… > Dependencies** from the project header.
 
-![See Dependencies](img/dependencies-open.gif)
+## Add dependencies
 
-To see all of your Project and Pipeline dependencies simply open the **Manage Dependencies** screen by going to **... >
-Options > Manage Dependencies**.
+When viewing the dependencies, you can also add new dependencies by selecting **Add Dependency**.
 
-### Add dependency
+| Parameter                   | Description                                                                                                                                                                                                                                                                             |
+| --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Scope                       | The dependency is enabled at the Project level or the Pipeline level.                                                                                                                                                                                                                   |
+| Type                        | The dependency is either from the Package Hub, Scala (Maven) or Python (PyPi).                                                                                                                                                                                                          |
+| Name                        | This will identify the dependency.                                                                                                                                                                                                                                                      |
+| Version/Package/Coordinates | For Package Hub dependencies, input the package version. For Scala, use the Maven coordinates in the `groupId:artifcatId:version` format. For example, use `org.postgresql:postgresql:42.3.3` For Python, use the package and the version number.                                       |
```

Comment:
I don't see the option to choose "Package Hub" when installing a **Pipelines** dependency. Can I provide a reason why?